.fixtures.yml

```diff
@@ -19,7 +19,9 @@ fixtures:
     certs:         "https://github.com/theforeman/puppet-certs.git"
     qpid:          "https://github.com/theforeman/puppet-qpid.git"
     pulp:          "https://github.com/theforeman/puppet-pulp.git"
-    pulpcore:      "https://github.com/theforeman/puppet-pulpcore.git"
+    pulpcore:
+      repo: "https://github.com/ekohl/puppet-pulpcore.git"
```

Comment:
This obviously needs to be converted in an updated minimum version in metadata.json.

Reply:
This needed to be changed before it was merged.

Reply:
(pushed 45e251bd10715cbc0d8fce4be67b0897e269b503 directly to master)